packages/opc-footer/README.md

```diff
@@ -14,24 +14,20 @@ npm install
 
 ## Usage
 
-the opc-footer have supports dark theme, by default the light theme is selected. for example
-footer with default theme:
+The opc-footer defaults to white background a.k.a light theme and supports dark theme as well.
```

Comment:
@hybridx can u follow same readme structure as opc-header for this component.

Reply:
Yes I'll add the change in the generator as well